TOC.md

```diff
@@ -7,7 +7,7 @@
     - [uploader](./solutions/file-uploader/) - uploader implementations for the most frequent cases
 
       - [regular](./solutions/file-uploader/regular/) - overall case
-      <!-- - [inline](./solutions/file-uploader/inline/) - without modal -->
+      - [inline](./solutions/file-uploader/inline/) - without modal
```

Comment:
please, change list order according other places

Reply:
Now it's similar everywhere.